sdk/typescript/src/providers/provider.ts

```diff
@@ -90,14 +105,16 @@ export abstract class Provider {
    * Get a map from module name to
    * structured representations of Move modules
    */
-  abstract getNormalizedMoveModulesByPackage(objectId: string,): Promise<SuiMoveNormalizedModules>;
+  abstract getNormalizedMoveModulesByPackage(
```

Comment:
is there some formatter I should use here?

Reply:
You should probably install the VScode Extension`Prettier - Code formatter`